docs/REST API/rest-api.md

```diff
@@ -826,6 +826,10 @@ None.
     "enable_sso_idp_login": false,
     "enable_jit_provisioning": false
   },
+  "conditional_access": {
+    "microsoft_entra_tenant_id": "<TENANT ID>",
+    "microsoft_entra_connection_configured": true
```

Comment:
@rachaelshaw does "microsoft_entra_connection_configured" add information here? Could we just use presence/absence of "microsoft_entra_tenant_id"?

Reply:
@jacobshandling We're saving a record of the tenant ID in the first step, before the user has done the microsoft admin consent, so we don't necessarily know if they've completed the configuration on Microsoft's side yet. Once the fleetdm.com proxy tells us they have, we store this boolean so we don't have to hit the fleetdm.com API every time we need that info.

Reply:
Okay, thanks for the clarification!